java-base/Dockerfile

```diff
@@ -17,13 +17,19 @@ LABEL name="Stackable image for OpenJDK" \
     summary="The Stackable OpenJDK base image." \
     description="This image is the base image for all Stackable Java product images."
 
-# We need to use EPEL, as openjdk 22 is not shipped with UBI9
-RUN rpm --install --replacepkgs https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+RUN cat <<EOF > /etc/yum.repos.d/adoptium.repo
```

Comment:
note: We might want to add a link to the docs where we took this snippet from: https://adoptium.net/en-gb/installation/linux/#_centosrhelfedora_instructions

Reply:
```suggestion
# See: https://adoptium.net/en-gb/installation/linux/#_centosrhelfedora_instructions
RUN cat <<EOF > /etc/yum.repos.d/adoptium.repo
```